src/commands/install.rs

```diff
@@ -296,30 +296,34 @@ impl Target {
     }
 
     fn command(&self, url: &str, dev: bool) -> Command {
-        match self {
+        let shell_command = match self {
             Target::Npm => {
-                let mut cmd = Command::new("npm");
-                cmd.arg("install");
+                let mut cmd = format!("npm install {url}");
                 if dev {
-                    cmd.arg("--save-dev");
+                    cmd.push_str(" --dev");
```

Comment:
This will format it as `npm install https://blah.io--dev` with a lacking space

Reply:
There's a space in front of the `--dev`, so I think this should format correctly.